File: codeforces-round-913-div3/A_rook.py
t = int(input())
letters = "abcdefgh"

for case in range(t):
    s = input()
    letter = s[0]
    number = s[1]

    for i in range(1, 9):
        if i != int(number):
            print(letter + str(i))

    for ch in letters:
        if ch != letter:
            print(ch + str(number))

File: old/codeforces-round-913-div3/a_rook.go
package main

import (
	// "bufio"
	"fmt"
	// "os"
	"strconv"
)

func main() {
	var t int
	fmt.Scan(&t)

	letters := "abcdefgh"

	for caseNum := 0; caseNum < t; caseNum++ {
		var s string
		fmt.Scan(&s)

		letter := string(s[0])
		number, _ := strconv.Atoi(string(s[1]))

		// Loop for numbers
		for i := 1; i <= 8; i++ {
			if i != number {
				fmt.Println(letter + strconv.Itoa(i))
			}
		}

		// Loop for letters
		for _, ch := range letters {
			if string(ch) != letter {
				fmt.Println(string(ch) + strconv.Itoa(number))
			}
		}
	}
}
